backend/src/test/java/org/cryptomator/hub/license/LicenseHolderTest.java

```diff
@@ -62,14 +68,16 @@ public void testValidDBTokenSet() {
 
 			holder.init();
 
-			Mockito.verify(validator, Mockito.times(1)).validate("token", "42");
+			Mockito.verify(validator, Mockito.times(2)).validate("token", "42");
 			Assertions.assertEquals(decodedJWT, holder.get());
 		}
 
 		@Test
 		@DisplayName("If database token is invalid, do net set it in license holder and nullify db entry")
 		public void testDBTokenOnFailedValidationNotSet() {
-			Mockito.when(validator.validate(Mockito.anyString(), Mockito.anyString())).thenThrow(JWTVerificationException.class);
+			Mockito.when(validator.validate(Mockito.anyString(), Mockito.anyString())).thenAnswer(invocationOnMock -> {
+				throw new JWTVerificationException("");
+			});
```

Comment:
Unnecessary change

Reply:
In the previous variant, I'm getting errors like 

```
org.opentest4j.AssertionFailedError: Unexpected exception type thrown, 
Expected :class com.auth0.jwt.exceptions.JWTVerificationException
Actual   :class com.auth0.jwt.exceptions.JWTVerificationException
<Click to see difference>


	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:67)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:35)
	at org.junit.jupiter.api.Assertions.assertThrows(Assertions.java:3083)
	at org.cryptomator.hub.license.LicenseHolderTest$TestSetter.testSetInvalidToken(LicenseHolderTest.java:159)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at io.quarkus.test.junit.QuarkusTestExtension.runExtensionMethod(QuarkusTestExtension.java:1015)
	at io.quarkus.test.junit.QuarkusTestExtension.interceptTestMethod(QuarkusTestExtension.java:829)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(InterceptingExecutableInvoker.java:103)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.lambda$invoke$0(InterceptingExecutableInvoker.java:93)
	.......
	at org.junit.platform.launcher.core.SessionPerRequestLauncher.execute(SessionPerRequestLauncher.java:53)
	at com.intellij.junit5.JUnit5IdeaTestRunner.startRunnerWithArgs(JUnit5IdeaTestRunner.java:57)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:232)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:55)
Caused by: com.auth0.jwt.exceptions.JWTVerificationException

```

Or just fails with 
![Screenshot from 2023-10-27 14-36-07](https://github.com/cryptomator/hub/assets/1786772/de96e3e4-56d4-409c-95ec-4fbd8deefd87)

I think here as well the `@PostConstruct` method is the problem.